### DISCUSSION TITLE
CSS & JS 라이브러리 코드 중복

### DISCUSSION THREAD

**Discussion by gwansikk** [KGU-CS-HOME/.github#7] [3. 이슈]
CSS 및 JS 라이브러리 관련해서 코드 중복이 너무 많습니다. 
페이지 및 컴포넌트 분리하며 코딩하다 보니 필요할 때마다 작성해서 발생된 것 같습니다. 한 페이지말고 다른 페이지에서도 사용될 거 같으면 상위(common_settings)로 올려주시고, 아래 리스트를 확인하시고 중복이라고 판단되면 삭제 부탁드립니다.

- sweetalert2
- jquery
- bootstrap
- ckeditor

삭제 하기 전 충분한 테스트 해주시고 판단하여 삭제해주세요. ^^

**Comment by gabrielyoon7** [comment]
굉장히 중요한 내용인 것 같습니다.

어느 정도 중복 사용이 된다고 판단되시면 그냥 common_settings로 넣으셔도 됩니다.
이 프로젝트에서 사용되는 라이브러리가 몇 개 없어서 로딩 시간에는 큰 차이가 없는 것 같습니다.
유지 보수 측면에서는 한 곳에 몰아 넣고 관리하는 것이 나은 것 같습니다.
